docs/metrics-guide/averaging-methods.md

```diff
@@ -0,0 +1,118 @@
+---
+subtitle: Macro, Micro, Weighted
+---
+
+# Averaging Methods
+
+For multiclass workflows like classification or object detection, metrics such as precision, recall, and F1-score are
+computed **per class**. To compute a single value that represents model performance across all classes, these per-class
+scores need to be aggregated. There are a few different averaging methods for doing this, most notably:
+
+- [**macro**](#macro-average): unweighted mean of all per-class scores
+- [**micro**](#micro-average): global average of per-sample TP, FP, FN scores
+- [**weighted**](#weighted-average): mean of all per-class scores, weighted by sample sizes for each class
+
+## Example: Multiclass Classification
+
+Let’s consider the following multiclass classification metrics, computed across a total of 10 samples:
+
+| Class | # Samples | <span title="# True Positives">TP</span> | <span title="# False Positives">FP</span> | <span title="# False Negatives">FN</span> | Precision | Recall | F1-score |
```

Comment:
I don't know why the tooltip for FN shows on TP but I feel like this is not something that can be done here:
<img width="800" alt="image" src="https://github.com/kolenaIO/kolena/assets/17771952/9708b501-ccd4-4206-a417-87aea00bb9fd">

Reply:
Argh, good find. Removed tooltips in favor of longer column names.